src/seedu/addressbook/AddressBook.java

```diff
@@ -199,9 +199,20 @@
      * ====================================================================
      */
     public static void main(String[] args) {
-        showWelcomeMessage();
-        processProgramArgs(args);
-        loadDataFromStorage();
+        showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
+        if (args.length >= 2) {
+		    showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
+		    exitProgram();
+		}
```

Comment:
As per our coding standard, indentation should be 4 spaces.
Use spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.